website/docs/guides/dremio-lakehouse.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Build a data lakehouse with dbt Core and Dremio Cloud
+id: build-dremio-lakehouse
+description:  Learn how to build a data lakehouse with dbt Core and Dremio Cloud.
+displayText: Build a data lakehouse with dbt Core and Dremio Cloud
+hoverSnippet: Learn how to build a data lakehouse with dbt Core and Dremio Cloud
+# time_to_complete: '30 minutes' commenting out until we test
+platform: 'dbt-core'
+icon: 'guides'
+hide_table_of_contents: true
+tags: ['Dremio', 'dbt Core']
+level: 'Intermediate'
+recently_updated: true
+---
+## Introduction
+
+This is a guide will demonstrate how to build a data lakehouse with dbt Core 1.5+ and Dremio Cloud. You can simplify and optimize your data infrastructure with dot’s robust transformation framework and Dremio’s open and easy data lakehouse. The integrated solution empowers companies to establish a strong data and analytics foundation, fostering self-service analytics and enhancing business insights while simplifying operations by eliminating the necessity to write complex ETL pipelines. 
```

Comment:
- sugg consistency in how we present versions (1.5+ or 1.5 and newer in other pages)
- i think 'dot' is meant to be 'dbt'?
- sugg spelling out let
- I'm prob overthinking but dbt is part of the ELT paradigm - do we not want to include that too? maybe that's a partnership team question

```suggestion
This guide will demonstrate how to build a data lakehouse with dbt Core 1.5 or new and Dremio Cloud. You can simplify and optimize your data infrastructure with dbt's robust transformation framework and Dremio’s open and easy data lakehouse. The integrated solution empowers companies to establish a strong data and analytics foundation, fostering self-service analytics and enhancing business insights while simplifying operations by eliminating the necessity to write complex Extract, transform, and load (ETL) pipelines. 
```